src/lib/mina_base/sparse_ledger.mli

```diff
@@ -30,49 +30,51 @@ val path_exn :
 
 val find_index_exn : t -> Account_id.t -> int
 
-val of_root :
-     depth:int
-  -> next_available_token:Token_id.t
-  -> next_available_index:int option
-  -> Ledger_hash.t
-  -> t
+val of_root : depth:int -> next_available_token:Token_id.t -> Ledger_hash.t -> t
+
+<<<<<<< HEAD
+val apply_user_command :
+||||||| a830646c1
+val has_locked_tokens_exn :
+  global_slot:Mina_numbers.Global_slot.t -> account_id:Account_id.t -> t -> bool
 
 val apply_user_command :
+=======
```

Comment:
I think you want `apply_user_command_exn` from the 2nd branch and `has _locked_tokens_exn` from the 1st branch.